### PR TITLE
adding compatibility with windows os

### PR DESCRIPTION
**Pull request by raskin-aleksandr**
Flow changed to be compatible with windows os

### CODE REVIEW

File: build/build_content/build_cli_flow.sl [hunk 1]
```diff
@@ -26,7 +28,7 @@ flow:
         default: 'target_dir + "/cloudslang-cli/cslang"'
         overridable: false
     - cloudslang_content_repo:
-        default: "'https://github.com/CloudSlang/cloud-slang-content.git'"
```
Comment (orius123) [inline build/build_content/build_cli_flow.sl]: why did you move to SSH?

**Comment by meirwah** [comment]
It seems that now on win version "cslang-cli.zip" is saved under the "cli" folder (instead of target)
@raskin-aleksandr @orius123 is that what you meant?

**Comment by orius123** [comment]
What. Ever works. Doesn't really matter.
